src/metkit/mars/Type.h

```diff
@@ -72,6 +71,15 @@ class Context {
 
 //----------------------------------------------------------------------------------------------------------------------
 
+class ITypeToByList {
```

Comment:
This looks fine. It could possibly be done by templating on a TypeT derivation. But I don't think that is significant.